### DISCUSSION TITLE
Is there any se3 function for mujoco.

### DISCUSSION THREAD

**Discussion by jeongsukchul** [google-deepmind/mujoco#992] [Asking for Help] [answered]
There is jacp function for mujoco but i can't find se3 function that gives me se3 matrix(for each body or end-effector) when the input is joint value.

**Comment by yuvaltassa** [comment]
You mean the rotation matrix? All the `mjData.something_xmat` is exactly that.

**Comment by yuvaltassa** [comment]
Or perhaps you mean the [motion axes](https://mujoco.readthedocs.io/en/latest/APIreference/APItypes.html#c-frame-variables)? Anyway your question is not very clear.

**Reply by jeongsukchul** [reply]
Im sorry. i can use mjData.something_xmat or others but which is only available when the present joint value is at that value
So I want "mj_something_mat" for not actually go to that position so i want this kinds of function
mj_mat(joint angle1, joint angle2 ...) so that when i put values about joint angle, i can get the something_mat without "actually move to that position" now is it clear?

**Reply by saran-t** [reply, marked as answer]
If I'm understanding your question correctly, you should be able to save the `qpos`, modify it, get whatever matrix you need (after calling `mj_kinematics`), then restore the saved qpos value.
